### PR TITLE
Quick(ish) fixes

### PR DESCRIPTION
**Pull request by thomasleese**
[Trello Card](https://trello.com/c/7XPMle03/1105-quickish-fixes-to-documentation)

### CODE REVIEW

File: config/tech-docs.yml [hunk 1]
```diff
@@ -2,15 +2,15 @@
 host: https://content-api.publishing.service.gov.uk
 
 # Header-related options
-show_govuk_logo: false
+show_govuk_logo: true
```
Comment (kevindew) [inline config/tech-docs.yml]: yay 👍

File: config/tech-docs.yml [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
 service_name: GOV.UK Content API
-service_link: https://content-api.publishing.service.gov.uk
+service_link: /
```
Comment (kevindew) [inline config/tech-docs.yml]: That's an improvement 👍

File: config/tech-docs.yml [hunk 3]
```diff
@@ -2,15 +2,15 @@
...
 phase: Work In Progress
 
 # Links to show on right-hand-side of header
 header_links:
-  About: index.html
-  Reference: reference.html
+  About: /index.html
```
Comment (kevindew) [inline config/tech-docs.yml]: can this also be `/` ?

File: source/index.html.md [hunk 4]
```diff
@@ -29,19 +29,19 @@ GOV.UK.
...
+ 2. Make a note of the path, for example: `/take-pet-abroad`
  3. Using a tool such as [curl](https://curl.haxx.se/),
     [Postman](https://www.getpostman.com/) or your web browser make a GET
     request to [`https://www.gov.uk/api/content/take-pet-abroad`](https://www.gov.uk/api/content/take-pet-abroad)
  4. You’ll receive a JSON response, the fields for this are
     explained in the [reference documentation](reference.html)
 
-For example, using [curl](https://curl.haxx.se/) command line utility and the
-[jq](https://stedolan.github.io/jq/) tool to display the response:
+For example, using [curl](https://curl.haxx.se/) command line utility tool to
```
Comment (kevindew) [inline source/index.html.md]: I'd probably cut the display the response part of this now we don't have jq (which was used to display the response)

**Comment by thomasleese** [comment]
@kevindew Comments addressed!

**Pull request merged**